evap/staff/tools.py

```diff
@@ -76,6 +78,16 @@ def create_user_list_html_string_for_message(users):
     return format_html_join("", "<br />{} {} ({})", ((user.first_name, user.last_name, user.email) for user in users))
 
 
+def append_user_list(message: str, user_profiles: Collection) -> SafeString:
```

Comment:
Maybe name it `append_user_list_if_not_empty` to make clear that an empty list will just end the sentence, which could lead to possibly confusing messages like:
`The following 0 users will be added to the evaluation.` (without an empty list)